### PR TITLE
Clean up docs `conf.py`

### PR DESCRIPTION
**Pull request by gunungpw**
- [x] Write a good description on what the PR does.
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

- clean up commented config from `python-docs-theme`
- add notes about use of default `furo` theme config

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -108,13 +108,7 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
```
Comment (Pierre-Sassoulas) [inline doc/conf.py:110]:
```suggestion
# documentation.
# html_theme_options = {}
```

File: doc/conf.py [hunk 2]
```diff
@@ -148,9 +142,6 @@
 
 # Custom sidebar templates, maps document names to template names.
 # Use Default Furo Sidebar
```
Comment (Pierre-Sassoulas) [inline doc/conf.py:144]:
```suggestion
# Currently we use the default Furo Sidebar
# html_sidebars = {}
```

**Review by Pierre-Sassoulas — APPROVED** [review]

File: doc/conf.py [hunk 3]
```diff
@@ -108,13 +108,8 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
```
Comment (Pierre-Sassoulas) [inline doc/conf.py:110]:
```suggestion
# documentation.
# Currently we use the default Furo configuration
```

File: doc/conf.py [hunk 4]
```diff
@@ -108,13 +108,8 @@
...
-# TO-DO: Disable thme options too see how Furo themes look in default config
-# html_theme_options = {
-#     "collapsiblesidebar": True,
-#     "issues_url": "https://github.com/pycqa/pylint/issues/new",
-#     "root_name": "PyCQA",
-#     "root_url": "https://meta.pycqa.org/en/latest/",
-# }
+# html_theme_options = {}
+# Use Furo theme default config
```
Comment (Pierre-Sassoulas) [inline doc/conf.py:112]:
```suggestion
```

**Comment by DanielNoord** [comment]
@Pierre-Sassoulas Should we just remove all the commented out theme options? These are out-generated from `Sphinx`. Imo, it's just clutter in the file.

**Comment by Pierre-Sassoulas** [comment]
>  Should we just remove all the commented out theme options? 

I think it's nice to be able to see the available options wthout having to read the Sphinx documentation if we need to modify the file. Maybe we could regenerate it from zero as it's very old by now.

**Pull request merged**